test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -230,14 +230,14 @@ static inline void multi_access_publisher(bool intra_process)
 
   rclcpp::executors::MultiThreadedExecutor executor;
 
-  auto pub = node->create_publisher<test_rclcpp::msg::UInt32>(node_topic_name);
+  const size_t iterations = 5 * executor.get_number_of_threads();
```

Comment:
Changed to `num_messages` (because it's the number of messages that we're going to send, and is used later in the code for that purpose) in b6035f5.